src/pages/case-studies.js

```diff
@@ -0,0 +1,381 @@
+        href: "https://www.youtube.com/watch?v=fNtG0iHYne4",
+      },
+      {
+        name: "Video 2",
+        href: "https://www.youtube.com/watch?v=vJaOKGWiyvU",
+      },
+    ],
+  },
+  {
+    logo: Wildlife,
+    description: [
+      "Wildlife Studios",
+      " uses eBPF through Cilium for high performance game networking",
+    ],
+    links: [
+      {
+        name: "Blog",
+        href: "https://cilium.io/blog/2020/09/03/wildlife-studios-multi-cluster-gaming-platform",
+      },
+    ],
+  },
+  {
+    logo: CaptialOne,
+    description: [
+      "Captial One",
+      " uses eBPF to secure their cloud infrastructure",
+    ],
+    links: [
+      {
+        name: "Video",
+        href: "https://www.youtube.com/watch?v=hwOpCKBaJ-w&ab_channel=eBPFSummit",
+      },
+    ],
+  },
+  {
+    logo: Sky,
+    description: ["Sky", " uses eBPF through Cilium for cloud networking"],
+    links: [
+      {
+        name: "Video",
+        href: "https://www.youtube.com/watch?v=u-4naOMfs_w",
+      },
+    ],
+  },
+  {
+    logo: Apple,
+    description: [
+      "Apple",
+      " uses eBPF through Falco for kernel security monitoring",
+    ],
+    links: [
+      {
+        name: "Video",
+        href: "https://www.youtube.com/watch?v=ZBlJSr6XkN8",
+      },
+    ],
+  },
+  {
+    logo: Walmart,
+    description: ["Apple", " uses eBPF for edge cloud load balancing"],
+    links: [
+      {
+        name: "Video 1",
+        href: "https://youtu.be/Fu4L8ewcO70",
+      },
+      {
+        name: "Video 2",
+        href: "https://youtu.be/thmAcyix8FM",
+      },
+    ],
+  },
+  {
+    logo: Palantir,
+    description: ["Palantir", " uses eBPF to debug networking problems"],
+    links: [
+      {
+        name: "Video",
+        href: "https://youtu.be/0RDp1IPxbg0",
+      },
+    ],
+  },
+  {
+    logo: Cruise,
+    description: ["Cruise", " uses eBPF to monitor GPU performance"],
+    links: [
+      {
+        name: "Video",
+        href: "https://youtu.be/7bdy2AkRjqE",
+      },
+    ],
+  },
+  {
+    logo: DigitalOcean,
+    description: ["Digital Ocean", " uses eBPF to monitor GPU performance"],
+    links: [
+      {
+        name: "Video",
+        href: "https://youtu.be/gcHxfhDT-I4",
+      },
+    ],
+  },
+  {
+    logo: Yahoo,
+    description: [
+      "Yahoo",
+      " uses eBPF for load balancing and performance analysis",
+    ],
+    links: [
+      {
+        name: "Video 1",
+        href: "https://youtu.be/-C86fBMcp5Q",
+      },
+      {
+        name: "Video 2",
+        href: "https://www.usenix.org/conference/lisa21/presentation/jones-zachary",
+      },
+    ],
+  },
+  {
+    logo: Bytedance,
+    description: ["Bytedance", " uses eBPF for system tracing"],
+    links: [
+      {
+        name: "Blog",
+        href: "https://inf.news/en/tech/3313635d67f796ae38e7ef64061bc7c4.html",
+      },
+    ],
+  },
+  {
+    logo: F5,
+    description: ["F5", " BIG-IP integration with Cilium VTEP feature"],
```

Comment:
It seems easier to insert `html` to the description than to use an array. 
```suggestion
    description: "<strong>F5</strong> BIG-IP integration with Cilium VTEP feature",
```